pkgs/tools/misc/mommy/default.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, writeText
+, shellspec
+, fetchpatch
+  # usage:
+  # pkgs.mommy.override {
+  #  mommySettings.sweetie = "catgirl";
+  # }
+  #
+  # $ mommy
+  # who's my good catgirl~
+, mommySettings ? null
+}:
+
+let
+  variables = lib.mapAttrs'
+    (name: value: lib.nameValuePair "MOMMY_${lib.toUpper name}" value)
+    mommySettings;
+  configFile = writeText "mommy-config" (lib.toShellVars variables);
+in
+stdenv.mkDerivation rec {
+  pname = "mommy";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "FWDekker";
+    repo = pname;
```

```suggestion
    repo = "mommy";
```

See: https://github.com/nix-community/nixpkgs-lint/issues/21

---

This is not applicable because this `mkDerivation` call is `rec`, not a lambda, so `overrideAttrs` will not internally propagate.

(And therefore a nitpick of minuscule proportions, functionality should be correct)

---

My bad, I think `finalAttrs` is preferred over `rec` now. Didn't catch that in the first review. Nitpick I agree.

See: https://github.com/NixOS/nixpkgs/pull/238775#discussion_r1235447008

---

631 `finalAttrs`, 9908 `rec`. still doesn't say much though